.gitignore

```diff
@@ -71,6 +71,8 @@ builds/
 #
 # example.tfvars
 **/terraform.tfvars
+**/backend.tf
+**/.terraform.lock.hcl
```

Comment:
terraform lock file file should be included in git and therefor *not* referenced in .gitignore.

Reply:
Done! change is included in the last commit.